documentation/intellij.asciidoc

```diff
@@ -22,3 +22,14 @@ You may also supply additional arguments as `devon intellij «args»`. These are
 |`ws-reverse-add`|reverse merge adding new properties
 |`create-script` |create launch script for this IDE, your current workspace and your OS
 |=======================
+
+There are variables that can be used for IntelliJ. These are explained by the following table:
+
+.Variables of devonfw-ide for intelliJ
+[options="header"]
+|=======================
+|*Variable*|*Default-Value*|*Meaning*
+|*`INTELLIJ_VERSION`*|`2019.2`|The version of the tool IntelliJ to install and use.
+|*`INTELLIJ_EDITION_TYPE`*|`C`|The edition of the tool IntelliJ to install and use. The value `C` mean Community edition and the value `U` mean Ultimate edition. The Ultimate edition requires a license. The user has to buy the license separately and it is not part of devon-ide. The devon-ide only supports download and installation.
```

Comment:
can you write `devonfw-ide` instead of `devon-ide` to be aligned with the officials. Thanks.

Reply:
I have corrected it with commit 3.